SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -151,6 +163,11 @@ bool CreatePlaceholders(Worker_SnapshotOutputStream* OutputStream)
 	checkf(PlaceholderCountAxis % 2 == 0, TEXT("The number of placeholders on each axis must be even."));
 	const float CHUNK_SIZE = 5.0f; // in SpatialOS coordinates.
 	int PlaceholderEntityIdCounter = SpatialConstants::PLACEHOLDER_ENTITY_ID_FIRST;
+
+	TArray<FString> ServerWorkerTypes;
+	Algo::Transform(GetDefault<USpatialGDKSettings>()->WorkerTypes.Array(), ServerWorkerTypes, [](const FName& element) { return element.ToString(); } );
```

Comment:
😍

Reply:
Yes, tencho asked me to introduce on in the slider PR that I'm touching up. This particular code will be removed though as I believe one of @tenevdev's PRs removes placeholders

Reply:
Why does removing placeholders prevent getting WorkerTypes from Settings working?